lms/envs/aws.py

```diff
@@ -601,6 +601,11 @@
     OAUTH_OIDC_ISSUER = ENV_TOKENS['OAUTH_OIDC_ISSUER']
     OAUTH_ENFORCE_SECURE = ENV_TOKENS.get('OAUTH_ENFORCE_SECURE', True)
     OAUTH_ENFORCE_CLIENT_SECURE = ENV_TOKENS.get('OAUTH_ENFORCE_CLIENT_SECURE', True)
+    # Defaults for the following are defined in lms.envs.common
+    if 'OAUTH_EXPIRE_DELTA_DAYS' in ENV_TOKENS:
+        OAUTH_EXPIRE_DELTA = datetime.timedelta(days=ENV_TOKENS['OAUTH_EXPIRE_DELTA_DAYS'])
+    if 'OAUTH_EXPIRE_DELTA_PUBLIC_DAYS' in ENV_TOKENS:
+        OAUTH_EXPIRE_DELTA_PUBLIC = datetime.timedelta(days=ENV_TOKENS['OAUTH_EXPIRE_DELTA_PUBLIC_DAYS'])
```

Comment:
@jibsheet These are still conditional, because the ENV_TOKENS value for each is an integer, not a timedelta, while the common.py version is a timedelta (that mimics the values in django-oauth2-provider)